docker/webui/run_openqa.sh

```diff
@@ -1,15 +1,40 @@
 #!/bin/bash
 
-# prepare environment
-if [ ! -f /data/db/db.sqlite ]; then
-  /usr/share/openqa/script/initdb --user geekotest --init_database
-else
-  /usr/share/openqa/script/upgradedb --user geekotest --upgrade_database
-fi
+function scheduler() {
+  su geekotest /usr/share/openqa/script/openqa-scheduler-daemon
```

Comment:
you use `su geekotest -c $cmd` further down, why differently here? Better be consistent

Reply:
solved